contribute/design/README.md

```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
+  - [openebs/cstor](https://github.com/openebs/cstor) is a fork of [openzfs/zfs](https://github.com/openzfs/zfs) (License: CDDL). This fork contains the code that modifies ZFS to run in user space. 
+- [openebs/istgt](https://github.com/openebs/istgt) contains the iSCSI Target functionality used by cStor volumes. This work is derived from earlier work available as FreeBSD port at http://www.peach.ne.jp/archives/istgt/. The original work was licensed under BSD license.
+- [openebs/cstor-csi](https://github.com/openebs/cstor-csi) is the CSI Driver for cStor Volumes. This will deprecate the external provisioners. Currently in beta. Dependencies are in: [go.mod](https://github.com/openebs/cstor-csi/blob/master/go.mod).
+- [openebs/cstor-operators](https://github.com/openebs/cstor-operators) contain the Kubernetes custom resources and operators to manage cStor Pools volumes. Currently in beta and used by cStor CSI Driver. This will replace the volume management functionality offered by OpenEBS API Server. Dependencies are in: [go.mod](https://github.com/openebs/cstor-operators/blob/master/go.mod).
+
+## Mayastor
+
+- [openebs/Mayastor](https://github.com/openebs/Mayastor) contains the Mayastor data engine, CSI driver and management utilitities. 
+  - [openebs/spdk](https://github.com/openebs/spdk) is a forked repository of [spdk/spdk](https://github.com/spdk/spdk) (License: BSD) for managing the upstream changes. 
+  - [openebs/spdk-sys](https://github.com/openebs/spdk-sys) (License: MIT) contains Rust bindings for SPDK. 
+  - [openebs/partition-identity](https://github.com/openebs/partition-identity) is forked from [pop-os/partition-identity](https://github.com/pop-os/partition-identity) (License: MIT) for managing the upstream changes. 
+  - [openebs/blkid](https://github.com/openebs/blkid) is forked from [pop-os/blkid](https://github.com/pop-os/blkid) (License: MIT) for managing the upstream changes
+  - [openebs/proc-mounts](https://github.com/openebs/proc-mounts/) is forked from [pop-os/proc-mounts](https://github.com/pop-os/proc-mounts) (License: MIT) for managing upstream changes
+  - [openebs/sys-mount](https://github.com/openebs/sys-mount/) is forked from [pop-os/sys-mount](https://github.com/pop-os/sys-mount) (License: MIT) for managing upstream changes
+  - [openebs/blkid-sys](https://github.com/openebs/blkid-sys/) is forked from [cholcombe973/blkid-sys](https://github.com/cholcombe973/blkid-sys) (License: MIT) for managing upstream changes
+
+
+
+## Dynamic Local PVs
+
+- [openebs/zfs-localpv](https://github.com/openebs/zfs-localpv) contains the CSI driver for provisioning Kubernetes Local Volumes on ZFS installed on the nodes. 
+- [openebs/rawfile-localpv](https://github.com/openebs/rawfile-localpv) contains the CSI driver for provisioning Kubernetes Local Volumes on Hostpath by creating sparse files. (Currently in Alpha). 
+
+
 
 # Further details
```

Comment:
I wonder if we should encourage people to submit a question or open an issue if they've read this far. Sort of like, giving them a template similar to this https://github.com/openebs/openebs/issues/new?template=feature-request.md 

The current document educates them, but does not give them a way to get involved in any of the projects. Maybe that's something we'll handle in a future tweak? Seeding the idea here, for now :)

Reply:
Good one @pgburt. Added a new section.